test/client/spec/components/boilerplate-component.spec.jsx

```diff
@@ -2,7 +2,8 @@
  * Client tests
  */
 import React from "react/addons";
-import Component from "src/components/boilerplate-component";
```

Comment:
Then name of _this_ file needs to mirror the app file, so I'm not sure this is a net improvement.